python/lsst/analysis/tools/actions/plot/histPlot.py

```diff
@@ -66,6 +73,21 @@ class HistPanel(Config):
         "is plotted, the percentile limit is the maximum value across all input data.",
         default=98.0,
     )
+    expectedValue = Field[float](
+        doc="Value at which to add a black solid vertical line.  Ignored if set to None.",
+        default=None,
+        optional=True,
+    )
+    histDensity = Field[bool](
+        doc="Whether to plot the histogram as a normalized probability distribution.  Must also "
+        "provide a value for expectedValue",
+        default=False,
+    )
+
+    def validate(self):
+        super().validate()
+        if self.histDensity and self.expectedValue is None:
+            raise FieldValidationError("Must provide expectedValue if histDensity is True")
```

Comment:
For me at least, this line of code fails, giving this error:
```
TypeError: FieldValidationError.__init__() missing 2 required positional arguments: 'config' and 'msg'
```
I'm not super familiar with this error, but grepping around the rest of the stack, it seems you need to raise it by passing in three args:
```
raise FieldValidationError(field, config, msg)
```